webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/RepositoryUpdater.scala

```diff
@@ -72,6 +74,24 @@ class RepositoryUpdater(
     log = log
   )
 
+  private val tempDirNamePrefix: String = "knora"
+
+  /**
+    * Deletes directories inside temp directory starting with `tempDirNamePrefix`.
+    */  
+  def deleteTempDirectories(): Unit = {
+    val rootDir = new File("/tmp/")
```

Comment:
I think adding that check doesn't make sense since [that part of code is commented out](https://github.com/dasch-swiss/dsp-api/blob/main/webapi/src/main/resources/application.conf#L341) and `settings.upgradeDownloadDir` [in this check](https://github.com/dasch-swiss/dsp-api/blob/main/webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/RepositoryUpdater.scala#L168) will always lead to None option, which goes then to `deleteTempDirectories()`.

Hmmm, is the `Paths.get()` suggestion because Windows uses backslashes?

Reply:
@subotic I've removed unused code in [881cf90](https://github.com/dasch-swiss/dsp-api/pull/2010/commits/881cf90f00717911850bb33bcbb4a1b7b4fe1822). I don't know why it wasn't commented out, but if my implementation will not work, it can be brought back. That means maybe instead of creating random folders, only one would be used.

I've tried out `Paths.get()` and `Path.of()`. Although all compiles, `test-repository-upgrade` hangs on last script `wait-for-knora.sh`. Once again, if this implementation somehow fail, idea described above, should make possible to use either `Paths.get()` or `Path.of()`.

Reply:
please don't merge a PR if the discussion is not finished.

The locally defined `docker-compose.yml` is irrelevant. The app is deployed in production with a different set of settings and another `docker-compose.yml`. Instead of adding 2 lines if code, you are now risking to break stuff in production. Not the right attitude in my opinion.

Reply:
I'm sorry but thought you are not going to reply and because already accepting this PR there isn't any critical issue here.

If you feel so, this changes could be reverted. But this isn't deployed directly to the production, we have TEST and STAGE too.

Reply:
Just checked PROD configuration and apparently `/tmp:/tmp` volume is also defined in `docker-compose.yml`, `/tmp/` folder exists and contains `knora*` update directories to remove.